src/hooks/broadcast.ts

```diff
@@ -55,4 +57,20 @@ const useUpdateBroadcast = (queryClient: QueryClient) =>
     }
   })
 
-export { useBroadcastDashboardQuery, useUpdateBroadcast, usePastBroadcastsQuery }
+const useLookupTemplateRecords = () =>
+  useQuery({
+    queryKey: ['lookupTemplateRecords'],
+    queryFn: getLookupTemplateRecords
+  })
+
+const useUpdateLookupTemplateRecord = (queryClient: QueryClient) =>
+  useMutation({
+    mutationFn: async ({ id, content }: { id: number; content: string }) => {
+      await updateLookupTemplateRecord(id, content)
+    },
+    onSuccess: async () => {
+      await queryClient.invalidateQueries({ queryKey: ['lookupTemplateRecords'] })
+    }
+  })
```

Comment:
It has a side effect so it should validate parameters itself
```suggestion
  useMutation({
    mutationFn: async ({ id, content }: { id: number; content: string }) => {
      if (!id || !content) {
      throw new Error('ID and content are required');
    }
      await updateLookupTemplateRecord(id, content)
    },
    onSuccess: async () => {
      await queryClient.invalidateQueries({ queryKey: ['lookupTemplateRecords'] })
    }
  })
```